tests/components/dyson/test_sensor.py

```diff
@@ -403,3 +307,49 @@ async def test_purecool_component_setup_only_once(devices, login, hass):
     await hass.async_block_till_done()
 
     assert len(hass.data[dyson.DYSON_SENSOR_DEVICES]) == 4
+
+
+@patch("libpurecool.dyson.DysonAccount.login", return_value=True)
+@patch(
+    "libpurecool.dyson.DysonAccount.devices",
+    return_value=[_get_purecool_device()],
+)
+async def test_dyson_purecool_filter_state_sensor(devices, login, hass):
+    """Test filter sensor with values."""
+    config = _get_config()
+    await async_setup_component(hass, dyson_parent.DOMAIN, config)
+    await hass.async_block_till_done()
+    discovery.load_platform(hass, "sensor", dyson_parent.DOMAIN, {}, config)
```

Comment:
This isn't needed. The Dyson sensor platform will be loaded when the Dyson integration is set up.